<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I2c_slave_gui</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openDevice">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Device</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxEventCfg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Event Config</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>339</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelEventAddress">
        <property name="text">
         <string>Slave Address #:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="setEvent">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="getEvent">
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEventType">
        <property name="text">
         <string>Event Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="eventType">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Read</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Write</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Read/Write</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="slaveAddressNumberEvent"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>271</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QPushButton" name="loadReply">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="reply"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelReply">
       <property name="text">
        <string>Reply:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>174</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="port"/>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBoxEnabled">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>341</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelGeneralCall">
       <property name="text">
        <string>General Call:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="generalCall">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSlaveAddressNumber">
       <property name="text">
        <string>Slave Address #:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="slaveAddressNumber"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="setAddress">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="getAddress">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="address"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
